### DISCUSSION TITLE
add site stackoverflow

### DISCUSSION THREAD

**Discussion by congaterori** [soxoj/maigret#134] [Ideas]
need add site stackoverflow

**Comment by soxoj** [comment]
Hi, thank you. I did small research earlier. As I understand, StackOverflow doesn't have profile URLs with username only without ID
The current format is:
```
https://stackoverflow.com/u/12341234 => https://stackoverflow.com/users/12341234/andrewahn
```
Details: https://meta.stackexchange.com/questions/22044/can-we-have-a-profile-url

Do you have an idea how we can get check account existence by username only?

**Comment by soxoj** [comment]
I found the way to do the fuzzy search (by username part) and implemented it in #144. Hope it will be useful for you.
